core/embed/prodtest/optiga_prodtest.c

```diff
@@ -127,9 +133,10 @@ void pair_optiga(void) {
   // it is OK for some of the intermediate operations to fail.
 
   uint8_t secret[SECRET_OPTIGA_KEY_LEN] = {0};
+  uint8_t secret_dec[SECRET_OPTIGA_KEY_LEN] = {0};
```

Comment:
The meaning of `secret` and `secret_dec` is identical. It's confusing to have two variables for the same thing. Let's just use `secret`.

Reply:
https://github.com/trezor/trezor-firmware/commit/7d6e4595ee680b60a373b82752d2835e6ae595ce

Reply:
removed https://github.com/trezor/trezor-firmware/commit/3e52219453a68f9a6b3b35ef40029aa589a9f230